### ISSUE TITLE
Fixing python version in shebangs

### ISSUE THREAD

**Issue by psaiz** [uhop/node-re2#121]
Hi,

The module deploys two python files with a shebang pointing to python (instead of python3). This creates some issues to create rpms. Would it be possible to change the shebangs to use python3 ?

Best regards, 
pablo

```
*** ERROR: ambiguous python shebang in ./node_modules/re2/vendor/re2/make_unicode_casefold.py: #!/usr/bin/python. Change it to python3 (or python2) explicitly.
*** ERROR: ambiguous python shebang in ./node_modules/re2/vendor/re2/make_unicode_groups.py: #!/usr/bin/python. Change it to python3 (or python2) explicitly.
```

**Comment by uhop** [comment]
`vendor` is a changing reference to the git repository [google/re2](https://github.com/google/re2/), which contains C++ and some other code. It is not part of `node-re2` codebase, but a separate project we do not control. I suggest proposing the suggested change there.

`node-re2` uses some C++ files from `google/re2` project to compile a binary extension. No python files, or any other files, are used for that.

**Comment by psaiz** [comment]
Hi Eugene, 
Thanks for looking into this. I've created the issue [there](https://github.com/google/re2/issues/350), and they are looking into it. If it gets accepted, would it be possible to have a new version of node-re2 with those changes?

Thanks again

**Comment by uhop** [comment]
>If it gets accepted, would it be possible to have a new version of node-re2 with those changes?

Of course. Please let me know, and I will issue a new technical release.

**Comment by psaiz** [comment]
Hi Eugene. 
Here I come again :-) 
The patch to google/re has been accepted. Would it be possible to get a new version of your module?

**Comment by uhop** [comment]
`node-re2` uses tagged releases of [google/re2](https://github.com/google/re2). IIRC, they usually release (tag) once a month on the 1<sup>st</sup>. The latest release was [2021-11-1](https://github.com/google/re2/releases/tag/2021-11-01), which is used by `node-re2` at this moment.

So as soon as they cut a release, I'll update and republish this project.

**Comment by psaiz** [comment]
Hi @uhop, there was indeed a new version of google/re2 on the 1st of the month. Would you mind creating a new one of yours?
Thanks again,

**Comment by uhop** [comment]
Published today as 1.17.4.
